activesupport/CHANGELOG.md

```diff
@@ -18,6 +18,11 @@
 
     *Akshay Vishnoi*
 
+*   Fix parsing JSON time in `YYYY-MM-DD hh:mm:ss` (without `Z`).
+    Before such time was considered in UTC timezone, now, to comply with standard, it uses local timezone.
+
+    *Grzegorz Witek*
+
```

Comment:
@arnvald Newly added CHANGELOG should be added at the top of the file. Please move this entry to the top of CHANGELOG file.